pr-data.csv

```diff
@@ -3163,7 +3163,7 @@ https://github.com/epam/eco-schema-catalog,7456ece21f9a88095885f20247189551da3a1
 https://github.com/Ericsson/ecchronos,5d58b0e4c922822c9df3d893fff2246d5784b11f,core,com.ericsson.bss.cassandra.ecchronos.core.repair.state.TestRepairStateImpl.testPartiallyRepaired,ID,Accepted,https://github.com/Ericsson/ecchronos/pull/294,
 https://github.com/Ericsson/ecchronos,5d58b0e4c922822c9df3d893fff2246d5784b11f,core,com.ericsson.bss.cassandra.ecchronos.core.repair.TestRepairGroup.testGetPartialRepairTasks,ID,Accepted,https://github.com/Ericsson/ecchronos/pull/244,
 https://github.com/Ericsson/ecchronos,5d58b0e4c922822c9df3d893fff2246d5784b11f,core,com.ericsson.bss.cassandra.ecchronos.core.repair.TestRepairTask.testPartialRepair,ID,Accepted,https://github.com/Ericsson/ecchronos/pull/244,
-https://github.com/Ericsson/ecchronos,5d58b0e4c922822c9df3d893fff2246d5784b11f,core,com.ericsson.bss.cassandra.ecchronos.core.repair.TestRepairTask.testRepairHalf,ID,,,
+https://github.com/Ericsson/ecchronos,5d58b0e4c922822c9df3d893fff2246d5784b11f,core,com.ericsson.bss.cassandra.ecchronos.core.repair.TestRepairTask.testRepairHalf,ID,DeveloperFixed,,https://github.com/Ericsson/ecchronos/commit/caad1754c527affdb35540c4fbbc4c2dd55e9751
```

Comment:
Please put `PR Link` https://github.com/Ericsson/ecchronos/pull/244 in the second-last column (rather than the commit in the last column).

Reply:
Edited, please check again